system/modules/timelog/partials/actions/timelogwidget.php

```diff
@@ -3,5 +3,37 @@
 function timelogwidget(\Web $w) {
 	
     $w->ctx("active_log", $w->Timelog->getActiveTimelogForUser());
-	
+    
+    $tracked_object = $w->Timelog->hasTrackingObject() ? $w->timelog->getTrackingObject() : null;
+    $info = '';
+    if (!empty($tracked_object)) {
+        //generate title and description
+        //check if task
+        if ($w->Timelog->getTrackingObjectClass() == 'Task') {
+            $info .= "<h3>Task [";
+            $info .= $tracked_object->id;
+            $info .= "]: ";
```

Comment:
Please remove this, this is what the template is used for! You should be sending the $tracked_object to the template and building this HTML there.